pymfe/statistical.py

```diff
@@ -1693,22 +1610,13 @@ def ft_w_lambda(cls,
            Principles of Data Mining and Knowledge Discovery (PKDD),
```

Comment:
I guess this reference is wrong.

Reply:
Hi Edesio. I've got this reference from our main reference paper (Adriano & Luis). Can you send me the corrected reference?

Reply:
For now, I think we can keep it. I will search more carefully for the appropriate reference.